Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,201 @@
+| Select element by class name             |`element(by.css('.some-css-class'))`           |`$('..some-css-class')`     |
+| Select element containing certain string Text |`element(by.cssContainingText(tag, text))`|`$('tag=text')`       |
+| Selecting Multiple Elements              |`element.all(by.css('.some-css-class'))`       |`$$('..some-css-class')`    |
+| Select element by ID                     |`element(by.css('#some-css-id'))`              |`$('#some-css-id')`        |
+| Select element inside a element with some class name |`element(by.css('some-css')).element(by.css('tag-within-css'))`  | `$('some-css').$('tag-within-css’)`      |
+| Selecting element one by one in loop     |`element.get(i)`                          |`$$(selector)[i]`     |
+
+There is no proper substitute available for the command `element.getWebElement()` in webdriverIO, but we can use `browser.findElement()` as a substitute for this.
+
+For example, take look at the following code of protractor:
+
+```js
+ await browser.executeScript(
+      'arguments[0].click()', await clickableElement.getWebElement());
+```
+
+WebdriverIO version will be like this:
+
+```js
+ await browser.execute(
+      'arguments[0].click()',
+      await browser.findElement('css selector', clickableElement));
+```
+
+## Events on selected elements
+
+| Events                                  | Protractor                               |WebdriverIO            |
+| -----------------------------------------| -----------------------------------------|-----------------------|
+| Click on the element                     |`element.click()`                         |`$(selector).click()`  |
+| Get the text content from a DOM-element  |`element.getText())`                      |`$(selector).getText()`|
+| Navigate to the given destination        |`browser.get(url)`                        |`browser.url(url)`     |
+| Get the url of the currently opened website.|`browser.getCurrentUrl()`              |`browser.getUrl()`     |
+| Get an attribute from a DOM-element based on the attribute name.|`element.getAttribute(attributeName)`|`$(selector).getAttribute(attributeName)`|
+| Wait until a given condition is fulfilled.|`browser.wait(condition, time, optional_msg)`|`browser.waitUntil(condition, { timeout, timeoutMsg, interval })`|
+| Send a sequence of keystrokes to the active element|`element.sendKeys(keys)` |`element.setValue(keys)`     |
+| Selected DOM-Element is clickable, visible, and exists.|`until.elementToBeClickable(element)` |`$(selector).isClickable()`|
+|Get the value of a `<textarea>`, `<select>` or `<input>` found by given selector|`element.getText()` |`$(selector).getValue()`|
+|Clear a `<textarea>` or `<input>` element’s value   |`element.clear()` |   `$(selector).clearValue()`  |
+
+For more details please visit [ webdriverio ](https://webdriver.io/docs/gettingstarted) official documentation page.
```

Comment:
```suggestion
For more details please visit [webdriverio's official documentation](https://webdriver.io/docs/gettingstarted).
```

For accessibility, it's best to make the linked text describe the what the link goes to

Reply:
Done.